aksel.nav.no/website/app/dev/_ui/Avatar.tsx

```diff
@@ -0,0 +1,108 @@
+import Image from "next/image";
+import { Children, ReactNode, isValidElement } from "react";
+import { BodyShort, HStack } from "@navikt/ds-react";
+import styles from "./Avatar.module.css";
+
+const MAX_AVATAR_COUNT = 30;
+
+export const avatarUrl = (avatar_id: string) => {
+  let _avatar_id = avatar_id;
+  if (!isNaN(parseInt(avatar_id))) {
+    _avatar_id =
+      `${parseInt(avatar_id) % MAX_AVATAR_COUNT}`.padStart(3, "0") ?? "broken";
+  }
+  return `/avatars/${_avatar_id}.svg`;
+};
+
+export type Avatar = {
+  imageSrc: string;
+  name: string;
+  description: string;
+};
+
+export const Avatar = ({
+  imageSrc,
+  name,
+  showName = false,
+}: {
+  /** URL to image for the avatar graphic. */
+  imageSrc: string;
+  /** Avatar must always have a name, as it becomes the image alt attribute. */
+  name: string;
+  /** The name of what the avatar represents. */
+  children?: ReactNode;
+  /** Whether or not to show the name next to the avatar. */
+  showName?: boolean;
+}) => {
+  return (
+    <HStack gap="space-4" align="center">
+      <Image
+        className={styles.avatarImage}
+        width="24"
+        height="24"
+        alt={`Avatar for ${name}`}
+        src={imageSrc}
+        priority
+        loading="eager"
+      />
+      {showName && (
+        <BodyShort className={styles.avatarName} size="small">
+          {name}
+        </BodyShort>
+      )}
```

Comment:
Showing both name and alt might be redundant for screen-readers.

Reply:
will toggle on aria-hidden for the BodyShort here? 🤔 (alt is always shown), or do we maybe not want "Avatar for..." if we want the name to be shown?